docker-compose-ci.yml

```diff
@@ -19,6 +19,9 @@ services:
     command: test --verbosity=2 --parallel=10 --no-input
     depends_on:
       - db
+    volumes:
+      - type: tmpfs
+        target: /tmp/gunicorn-workers
```

Comment:
@jedef shouldn't we have the same folder here as in Dockerfile line 12?
`/tmp/gunicorn_workers`

Reply:
Yes good catch I fixed that.